### DISCUSSION TITLE
sygil-webui\horde_bridge.cmd make Error prompt"txt2img() got an unexpected keyword argument 'karras'"

### DISCUSSION THREAD

**Discussion by jgdq** [hlky/stable-diffusion-webui#1691] [Q&A]
some displayed.
DEBUG      @ 2022-11-25 00:24:28 | __main__:run_bridge:2712 - Server https://stablehorde.net has no valid generations to do for us. Skipped Info: {'untrusted': 9, 'models': 39}.
DEBUG      @ 2022-11-25 00:26:46 | __main__:run_bridge:2716 - Request with id ab2a571e-ed00-4ba8-82c8-24808b37d6ff picked up. Initiating work...
ERROR      @ 2022-11-25 00:26:46 | __main__:<module>:2839 - An error has been caught in function '<module>', process 'MainProcess' (28212), thread 'MainThread' (21416):
> File "scripts\webui.py", line 2839, in <module>
    run_bridge(1, horde_api_key, horde_name, horde_url, horde_priority_usernames, horde_max_pixels, horde_nsfw, horde_censor_nsfw, horde_blacklist, horde_censorlist)
  File "scripts\webui.py", line 2735, in run_bridge
    images, seed, info, stats = txt2img(**current_payload)
                                │         └ {'prompt': 'sunset on a lush tropical island beach vivid colorful mystical', 'ddim_steps': 50, 'n_iter': 1, 'sampler_name': '...
                                └ <function txt2img at 0x00000182A0EF7DC0>

TypeError: txt2img() got an unexpected keyword argument 'karras'
Relauncher: Process is ending. Relaunching in 1s...

**Comment by db0** [comment]
Do not use the bridge in this repo. Please use https://github.com/Sygil-Dev/nataili
